tests/android/app/build.gradle

```diff
@@ -1,10 +1,13 @@
+// Add the Firebase Crashlytics plugin.
+
 apply plugin: 'com.android.application'
-apply plugin: 'io.fabric'
+apply plugin: 'com.google.firebase.crashlytics'
```

Comment:
don't forget to move the google-services plugin to just under com.android.application - that's a hunch to test as it is the upstream current documentation and we just altered our android integration documentation to match it. It may be why I'm compile-erroring in my throwaway demonstrator and the tests app isn't? At the least though it will make the tests app conform to current rnfirebase.io docs

Reply:
Done 😄 . Historically, it appears it was necessary to put this at the bottom ([ref](https://stackoverflow.com/a/42062624) & [ref](https://stackoverflow.com/q/55539495)) of the app `build.gradle`, the references to this phenomenon appear to have been removed from Android docs however, and Firebase docs have it nested under `com.android.application` plugin.